PENDING_RELEASE_NOTES.md

```diff
@@ -1,7 +1,8 @@
-## Version 17.7.1
+## Version 17.9.0
```

Comment:
Might need to merge from upstream/development again. I don't think there should be any PENDING notes on development, and probably never again, assuming the deployment scripts work

Wouldn't hurt to update the complete RELEASE NOTES too, if you can figure out what was actually released